packages/web3-eth/src/web3_eth.ts

```diff
@@ -219,7 +220,8 @@ export class Web3Eth extends Web3Context<Web3EthExecutionAPI, RegisteredSubscrip
 	 * ```
 	 */
 	public async getHashRate<ReturnFormat extends DataFormat = typeof DEFAULT_RETURN_FORMAT>(
-		returnFormat: ReturnFormat = DEFAULT_RETURN_FORMAT as ReturnFormat,
+		returnFormat: ReturnFormat = (this.defaultReturnFormat ??
+			DEFAULT_RETURN_FORMAT) as ReturnFormat,
```

Comment:
```suggestion
			web3Context.defaultReturnFormat) as ReturnFormat,
```
for all places as now we have updatable default set in context so that should be used instead of `DEFAULT_RETURN_FORMAT` from types package, so:
if some one is explicitly setting format via this.defaultReturnFormat it will work, but if not it will send DEFAULT_RETURN_FORMAT to rpc-wrapper, so rpc-wrapper will get value and will not use context default?